Source/WebCore/style/StyleTreeResolver.cpp

```diff
@@ -937,7 +937,7 @@ const RenderStyle* TreeResolver::existingStyle(const Element& element)
     return style;
 }
 
-auto TreeResolver::updateStateForQueryContainer(Element& element, const RenderStyle* style, ContainerType previousContainerType, Change& change, DescendantsToResolve& descendantsToResolve) -> QueryContainerAction
+auto TreeResolver::updateStateForQueryContainer(Element& element, const RenderStyle* style, const RenderStyle* existingStyle, Change& change, DescendantsToResolve& descendantsToResolve) -> QueryContainerAction
```

Comment:
nit: usually we put old before new, but I don't feel too strongly

Reply:
who is "we" in this?

Reply:
I often see `oldStyle` before `newStyle` in the codebase